model/src/main/proto/spine_examples/chatspn/chat/chat.proto

```diff
@@ -60,13 +60,17 @@ message Chat {
     // Members with the appropriate permissions can add new users to the group chat.
     // Members can leave a group chat, and the chat is only deleted when the last member has left.
     //
-    ChatType type = 3;
+    ChatType type = 3 [(required) = true];
 
     enum ChatType {
-        CT_PERSONAL = 0;
-        CT_GROUP = 1;
+        CT_UNSPECIFIED = 0;
```

Comment:
I think it would be good to explain the meaning of this field in the documentation.

Reply:
@Artem-Semenov-dev I think it's fine not to have any documentation on this topic here, as it is expected that engineers [do their RTFMs](https://protobuf.dev/programming-guides/proto3/#enum).